packages/inter-protocol/src/proposals/utils.js

```diff
@@ -81,7 +81,7 @@ const provideWhen = async (store, key, make) => {
 
 /**
  *
- * @param {{ scratch: ERef<MapStore<string, CopyTagged>> }} homeP
+ * @param {{ scratch: ERef<Map<string, CopyTagged>> }} homeP
```

Comment:
Does `scratch` actually have `Map` semantics? If not that'd be a very unobvious cast that hide the runtime differences.

Reply:
Yes, it does https://github.com/Agoric/agoric-sdk/blob/bf9b05ffef83d26a98df53953ee231bca637ae5b/packages/solo/src/scratch.js#L17-L20